source/renderer/zettlr-editor.js

```diff
@@ -222,6 +222,39 @@ class ZettlrEditor {
 
     this._searcher.setInstance(this._cm)
 
+    this._cm.on('keyup', (cm, event) => {
+      if (event.altKey &&
+        event.code === 'Enter' &&
+        !event.ctrlKey &&
+        !event.metaKey) {
+        let cur = cm.getCursor()
+        let linkStart = cm.getOption('zkn').linkStart
+        let linkEnd = cm.getOption('zkn').linkEnd
+        let line = cm.getLine(cur.line)
+
+        let linkStartPos = line.indexOf(linkStart)
+        let linkEndPos = line.indexOf(linkEnd)
+
+        if (linkStartPos !== -1 &&
+          linkEndPos !== -1 &&
+          linkStartPos < linkEndPos &&
+          (linkStartPos - 1) <= cur.ch &&
+          (linkEndPos + linkEnd.length) >= cur.ch) {
+          let linkName = line.substr(linkStartPos + linkStart.length, linkEndPos - linkStartPos - linkStart.length)
+          if (event.shiftKey) {
+            // Performa a save to avoid a problem where a file
+            // is saved while trying to create or open a link.
+            // That action cancels the file open/create
+            this._renderer.saveFile()
+            this.markClean(this._currentHash)
+            this._renderer.openOrCreate(linkName)
```

Comment:
As stated, this function needs to go, as the function below (`autoSearch`) will implement that behaviour once I've done my part! The plus side of this is that we can get rid of this whole if/else-thingy.

Oh, plus I realised we don't even have to add new shortcuts in that case o_O

Oh god, please forgive me, I'm so utterly absent-minded at the moment … I'll just leave this comment here, and wait for your reply on that matter … 🙈